monitors/archive_monitor/isis_archive_monitor_helper.py

```diff
@@ -0,0 +1,48 @@
+"""
+File to store messages and data relating to ISIS_archive_monitor
+"""
+
+# ================================= Data ======================================= #
+
+LOG_FORMAT = '%(asctime)s : %(message)s'
+STOMP_LOG_FORMAT = '%(asctime)s :      %(message)s'
+VALID_INST = ['GEM', 'POLARIS', 'WISH', 'MUSR', 'OSIRIS']
```

Comment:
Do we not have a list of valid instruments elsewhere in the codebase. I would be tempted to place this in a central location for other validators (present or future)

Reply:
I agree but this may form part of the larger discussion regarding settings in autoreduction which we had on Wednesday. We should decide what our structure for settings should look like first then move these there (could well be part of a separate PR depending on when that decision is made)